.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,35 @@
+# Contribution guide
+
+Everyone is welcome to contribute to SUMO Live Helper. If you have questions, you
+can reach us by joining #sumo-livehelper on 
+[chat.mozilla.org](https://chat.mozilla.org).
```

Comment:
Should the link be https://chat.mozilla.org/#/room/#sumo-livehelper:mozilla.org since that will open the correct room rather than just the general chat page?

Reply:
I think will be interesting a direct link... 🤔

Reply:
I think is interesting also take a look at [this repository](https://github.com/mozilla-mobile/fenix) as a "north" (I liked the organization of the sections PR and Getting Involved)

Reply:
Oh good idea! I totally forgot it was possible to do so with matrix :). Done.